packs/journals/classes.json

```diff
@@ -456,7 +456,7 @@
.spells-srd.Item.Spirit Link]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Protection</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Blood Ward]</em> hex; your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Mystic Armor]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Vengeance</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Needle of Vengeance]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Phantom Pain]</em>.</li></ul><p><strong>GREATER LESSONS</strong></p><p>You can select from these lessons when a feat or other effect grants you a greater lesson.</p><ul><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Mischief</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Deceiver's Cloak]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Mad Monkeys]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Shadow</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Malicious Shadow]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Chilling Darkness]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Snow</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Personal Blizzard]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Wall of Wind]</em>.</li></ul><p><strong>MAJOR LESSONS</strong></p><p>You can select from these lessons when a feat or other effect grants you a major lesson.</p><ul><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Bargains</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Over the Coals]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Subconscious Suggestion]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Death</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Curse of Death]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Raise Dead]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Renewal</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Restorative Moment]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Field of Life]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of the Frozen Queen</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Glacial Heart]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Wall of Ice]</em>.</li></ul>",
```

Comment:
```suggestion
                "content": "<p><em>You command powerful magic, not through study or devotion to any ideal, but as a vessel or agent for a mysterious, otherworldly patron that even you don't entirely understand. This entity might be a covert divinity, a powerful fey, a manifestation of natural energies, an ancient spirit, or any other mighty supernatural being-but its nature is likely as much a mystery to you as it is to anyone else. Through a special familiar, your patron grants you versatile spells and powerful hexes to use as you see fit, though you're never certain if these gifts will end up serving your patron's larger plan.</em></p><p><strong>Key Attribute: INTELLIGENCE</strong></p><p>At 1st level, your class gives you an ability boost to Intelligence.</p><p><strong>Hit Points: 6 plus your Constitution modifier</strong></p><p>You increase your maximum number of HP by this number at 1st level and every level thereafter.</p><h1 class=\"title\">Roleplaying the Witch</h1><h2 class=\"title\">During Combat Encounters...</h2><p>You cast spells to change the course of battle. You use magical hexes to hamper enemies and aid allies, while leveraging more powerful spells to control the battlefield, heal, or harm, aided by your extraordinary familiar, brewed potions, and magical items.</p><h2 class=\"title\">During Social Encounters...</h2><p>You provide knowledge on numerous topics, including a variety of magical matters, and you might call upon your patron's magic to charm or deceive others.</p><h2 class=\"title\">While Exploring...</h2><p>You remain alert for magical traps and treasures, employing a clever array of spells to overcome obstacles that stand in your way. Your familiar might aid you through its own considerable set of exceptional abilities.</p><h2 class=\"title\">In Downtime...</h2><p>You brew potions, craft other magical items, or hunt for new spells for your familiar to learn. You might try to learn more about your patron, their aims, or your own powers, and you might seek out the company of other witches for collaboration or community.</p><h2 class=\"title\">You Might...</h2><ul><li>Strive to learn more about your patron or familiar, your patron's goals, why they chose to empower you, and how you fit into their plans.</li><li>Seek out new sources of magic, like scrolls and spellbooks, to supplement the spells your patron provides.</li><li>View your familiar as a steadfast ally, a dear friend, or a necessary nuisance, depending on its personality.</li></ul><h2 class=\"title\">Others Probably...</h2><ul><li>Wonder about the nature of your patron and the source of your magic, worrying you'll turn on them or that you unknowingly serve to a foul power.</li><li>Appreciate your ability to aid them with magic, whether you do so by helping them directly or by hampering their adversaries.</li><li>Take care not to offend you, fearful that you'll place a malicious hex on them if you're angered.</li></ul><h1 class=\"title\">Initial Proficiencies</h1><p>At 1st level, you gain the listed proficiency ranks in the following statistics. You are untrained in anything not listed unless you gain a better proficiency rank in some other way.</p><h2 class=\"title\">Perception</h2><p>Trained in Perception</p><h2 class=\"title\">Saving Throws</h2><p>Trained in Fortitude</p><p>Trained in Reflex</p><p>Expert in Will</p><h2 class=\"title\">Skills</h2><p>Trained in one skill determined by your patron</p><p>Trained in a number of additional skills equal to 3 plus your Intelligence modifier</p><h2 class=\"title\">Attacks</h2><p>Trained in simple weapons</p><p>Trained in unarmed attacks</p><h2 class=\"title\">Defenses</h2><p>Untrained in all armor</p><p>Trained in unarmored defense</p><h2 class=\"title\">Spells</h2><p>Trained in spell attack modifier</p><p>Trained in spell DC</p><h2>Class DC</h2><p>Trained in witch class DC</p><h1 class=\"title\">Class Features</h1><p>You gain these features as a Witch. Abilities gained at higher levels list the levels at which you gain them next to the features' names.</p><table class=\"pf2-table\"><thead><tr><th>Your Level</th><th>Class Features</th></tr></thead><tbody><tr><td>1</td><td>Ancestry and background, initial proficiencies, patron, familiar, witch spellcasting, hexes</td></tr><tr><td>2</td><td>Skill feat, witch feat</td></tr><tr><td>3</td><td>2nd-rank spells, general feat, skill increase</td></tr><tr><td>4</td><td>Skill feat, witch feat</td></tr><tr><td>5</td><td>3rd-rank spells, attribute boosts, ancestry feat, magical fortitude, skill increase</td></tr><tr><td>6</td><td>Familiar ability, skill feat, witch feat</td></tr><tr><td>7</td><td>4th-rank spells, expert spellcaster, general feat, skill increase</td></tr><tr><td>8</td><td>Skill feat, witch feat</td></tr><tr><td>9</td><td>5th-rank spells, ancestry feat, reflex expertise, skill increase</td></tr><tr><td>10</td><td>Attribute boosts, skill feat, witch feat</td></tr><tr><td>11</td><td>6th-rank spells, general feat, perception expertise, skill increase, weapon expertise</td></tr><tr><td>12</td><td>Familiar ability, skill feat, witch feat</td></tr><tr><td>13</td><td>7th-rank spells, ancestry feat, defensive robes, skill increase, weapon specialization</td></tr><tr><td>14</td><td>Skill feat, witch feat</td></tr><tr><td>15</td><td>8th-rank spells, attribute boosts, general feat, master spellcaster, skill increase</td></tr><tr><td>16</td><td>Skill feat, witch feat</td></tr><tr><td>17</td><td>9th-rank spells, ancestry feat, skill increase, will of the pupil</td></tr><tr><td>18</td><td>Familiar ability, skill feat, witch feat</td></tr><tr><td>19</td><td>General feat, legendary spellcaster, patron's gift, skill increase</td></tr><tr><td>20</td><td>Attribute boosts, skill feat, witch feat</td></tr></tbody></table><h2 class=\"title\">Ancestry and Background</h2><p>In addition to the abilities provided by your class at 1st level, you have the benefits of your selected ancestry and background.</p><h2 class=\"title\">Initial Proficiencies</h2><p>At 1st level, you gain a number of proficiencies that represent your basic training. These proficiencies are noted at the start of this class.</p><h2 class=\"title\">Patron</h2><p>You weren't born with the power to cast spells, nor have you spent years in devotion to tomes, deities, or mystical secrets. Your power comes through a potent being that has chosen you as their vessel to carry forth some agenda in the world. This entity is typically mysterious and distant, revealing little of their identity and motivations, and they grant you spells and other magical powers through a familiar, which serves as a conduit for their power.</p><p>A patron might be a deity or demigod, a coven of powerful hags, a fey lord, an archdevil, or a similarly powerful entity, or perhaps multiple such figures working in tandem. As you gain more of your patron's power, you might learn more about who or what they are-certain combinations of themes and lessons suggest particular patrons or agendas-but patrons empower witches for their own secretive reasons, which they rarely reveal in full.</p><p>At 1st level, choose your patron's theme, which determines your spellcasting tradition, a skill, a special cantrip you gain, and a spell added to your familiar.</p><h2 class=\"title\">Witch Spellcasting</h2><p>Using your familiar as a conduit, your patron provides you the power to cast spells. You can cast spells using the @UUID[Compendium.pf2e.actionspf2e.Item.Cast a Spell] activity, and you can supply material, somatic, and verbal components when casting spells.</p><p>At 1st level, you can prepare up to two 1st-level spells and five cantrips each morning from the spells your familiar knows. Prepared spells remain available to you until you cast them or until you prepare your spells again. The number of spells you can prepare is called your spell slots.</p><p>As you increase in level as a witch, your number of spell slots and the highest level of spells you can cast from spell slots increase, as shown in Table 2-7: Witch Spells per Day.</p><p>Some of your spells require you to attempt a spell attack roll to see how effective they are, or have your enemies roll against your spell DC (typically by attempting a saving throw). Since your key ability is Intelligence, your spell attack rolls and spell DCs use your Intelligence modifier.</p><h3 class=\"title\"><strong>Heightening Spells</strong></h3><p>When you get spell slots of 2nd level and higher, you can fill those slots with stronger versions of lower-level spells. This increases the spell's level, heightening it to match the spell slot. Many spells have specific improvements when they are heightened to certain levels.</p><table class=\"pf2-table\"><thead><tr><th>Your Level</th><th>Cantrips</th><th>1st</th><th>2nd</th><th>3rd</th><th>4th</th><th>5th</th><th>6th</th><th>7th</th><th>8th</th><th>9th</th><th>10th</th></tr></thead><tbody><tr><td>1</td><td>5</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>2</td><td>5</td><td>3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>3</td><td>5</td><td>3</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>4</td><td>5</td><td>3</td><td>3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>5</td><td>5</td><td>3</td><td>3</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>6</td><td>5</td><td>3</td><td>3</td><td>3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>7</td><td>5</td><td>3</td><td>3</td><td>3</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>8</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>9</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>10</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>11</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>12</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>13</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>2</td><td>-</td><td>-</td><td>-</td></tr><tr><td>14</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>-</td><td>-</td><td>-</td></tr><tr><td>15</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>2</td><td>-</td><td>-</td></tr><tr><td>16</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>-</td><td>-</td></tr><tr><td>17</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>2</td><td>-</td></tr><tr><td>18</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>-</td></tr><tr><td>19</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>1*</td></tr><tr><td>20</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>1*</td></tr><tr><td style=\"color:white;width:120px;background-color:#522e2c;text-align:center;height:25px\" colspan=\"12\">* The patron conduit class feature gives you a 10th-level spell slot that works a bit differently from other spell slots.</td></tr></tbody></table><h3 class=\"title\"><strong>Cantrips</strong></h3><p>A cantrip is a special type of spell that doesn't use spell slots. You can cast a cantrip at will, any number of times per day. A cantrip is always automatically heightened to half your level rounded up-this is usually equal to the highest level of spell you can cast as a witch. For example, as a 1st-level witch, your cantrips are 1st-level spells, and as a 5th-level witch, your cantrips are 3rd-level spells.</p><h3><strong>Familiar</strong></h3><p>Your patron has sent you a familiar, a mystical creature that teaches you and facilitates your spells. This familiar follows the basic rules for familiars, though as it's a direct conduit between you and your patron, it's more powerful than other familiars. Your familiar gains an extra familiar ability, and gains another extra ability at 6th, 12th, and 18th levels.</p><p><strong>Spells:</strong> Your familiar is the source and repository of the spells your patron has bestowed upon you, and you must commune with your familiar to prepare your spells each day using your witch spellcasting. Your familiar starts off knowing 10 cantrips, five 1st-level spells, and one additional spell determined by your patron's theme. You choose these spells from the common spells of the tradition determined by your patron or from other spells of that tradition you gain access to.</p><p>Each time you gain a level, your patron teaches your familiar two new spells of any level you can cast, chosen from common spells of your tradition or others you gain access to. Feats can also grant your familiar additional spells.</p><p><strong>Learning Spells:</strong> Your familiar can learn new spells independently of your patron. It can learn any spell on your tradition's spell list by physically consuming a scroll of that spell in a process that takes 1 hour. You can use the @UUID[Compendium.pf2e.actionspf2e.Item.Learn a Spell] exploration activity to prepare a special written version of a spell, which your familiar can consume as if it were a scroll. You and your familiar can use the Learn a Spell activity to teach your familiar a spell from another witch's familiar. Both familiars must be present for the entirety of the activity, the spell must be on your spellcasting tradition's spell list, and you must pay the usual cost for that activity, typically in the form of an offering to the other familiar's patron. You can't prepare spells from another witch's familiar.</p><p><strong>Undying</strong>: If your familiar dies, your patron replaces it during your next daily preparations. The new familiar might be a duplicate or reincarnation of your former familiar or a new entity altogether, but it knows the same spells your former familiar knew regardless. Your familiar's death doesn't affect any spells you have already prepared.</p><h2 class=\"title\">Hexes</h2><p>Your patron and familiar teach you special spells called hexes. A hex is a short-term effect drawn directly from your patron's magic. As such, you can cast only one hex each turn; attempts to cast a second hex spell on that turn fail and the spellcasting actions are lost.</p><p>Hexes are a type of focus spell. It costs 1 Focus Point to cast a focus spell, and you start with a focus pool of 1 Focus Point. You refill your focus pool during your daily preparations, and you can regain 1 Focus Point by spending 10 minutes using the @UUID[Compendium.pf2e.actionspf2e.Item.Refocus] activity to commune with your familiar.</p><p>Focus spells are automatically heightened to half your level rounded up. Focus spells don't require spell slots, and you can't cast them using spell slots. Certain feats can give you more focus spells and increase the size of your focus pool, though your focus pool can never hold more than 3 Focus Points.</p><p>You learn the <em>@UUID[Compendium.pf2e.spells-srd.Item.Phase Familiar]</em> hex, which you can cast as a reaction to protect your familiar from harm. You learn most other hexes from witch lessons.</p><p><strong>Hex Cantrips</strong></p><p>Hex cantrips are special hexes that don't cost Focus Points, so you can cast them as often as you like, though you can still cast only one hex each round. Hex cantrips are in addition to the cantrips you choose with your witch spellcasting and aren't counted toward your prepared cantrips. Your hex cantrips are determined by your patron theme.</p><h2 class=\"title\">Skill Feats<span style=\"float:right\">Level 2</span></h2><p>At 2nd level and every 2 levels thereafter, you gain a skill feat. You must be trained or better in the corresponding skill to select a skill feat.</p><h2 class=\"title\">Witch Feats<span style=\"float:right\">Level 2</span></h2><p>At 2nd level and every even-numbered level, you gain a witch class feat.</p><h2 class=\"title\">General Feats<span style=\"float:right\">Level 3</span></h2><p>At 3rd level and every 4 levels thereafter, you gain a general feat.</p><h2 class=\"title\">Skill Increases<span style=\"float:right\">Level 3</span></h2><p>At 3rd level and every 2 levels thereafter, you gain a skill increase. You can use an increase to either become trained in one skill you're untrained in, or to increase your proficiency rank in one skill in which you're already trained to expert.</p><p>At 7th level, you can use skill increases to become a master in a skill in which you're already an expert, and at 15th level, you can use them to become legendary in a skill in which you're already a master.</p><h2 class=\"title\">Attribute Boosts<span style=\"float:right\">Level 5</span></h2><p>At 5th level and every 5 levels thereafter, you boost four different ability scores. You can use these ability boosts to increase your ability scores above 18. Boosting an ability score increases it by 1 if it's already 18 or above, or by 2 if it starts out below 18.</p><h2 class=\"title\">Ancestry Feats<span style=\"float:right\">Level 5</span></h2><p>In addition to the ancestry feat you started with, you gain an ancestry feat at 5th level and every 4 levels thereafter.</p><h2 class=\"title\">Magical Fortitude<span style=\"float:right\">Level 5</span></h2><p>Magical power has improved your body's resiliency. Your proficiency rank for Fortitude saves increases to expert.</p><h2 class=\"title\">Expert Spellcaster<span style=\"float:right\">Level 7</span></h2><p>Your inherent magic responds easily and powerfully to your command. Your proficiency ranks for spell attack rolls and spell DCs for your witch spellcasting increase to expert.</p><h2 class=\"title\">Reflex Expertise<span style=\"float:right\">Level 9</span></h2><p>Your reflexes are lightning fast, helping you to avoid danger. Your proficiency rank for Reflex saves increases to expert.</p><h2 class=\"title\">Perception Expertise<span style=\"float:right\">Level 11</span></h2><p>You remain alert to threats around you. Your proficiency rank for Perception increases to expert.</p><h2 class=\"title\">Weapon Expertise<span style=\"float:right\">Level 11</span></h2><p>Through sheer experience, you've improved your technique with your weapons. Your proficiency ranks for simple weapons and unarmed attacks increase to expert.</p><h2 class=\"title\">Defensive Robes<span style=\"float:right\">Level 13</span></h2><p>The flow of magic through your spellcasting and your defensive training combine to help you get out of the way before an attack. Your proficiency rank in unarmored defense increases to expert.</p><h2 class=\"title\">Weapon Specialization<span style=\"float:right\">Level 13</span></h2><p>You can inflict greater injuries with the weapons you know. You deal 2 additional damage with weapons and unarmed attacks in which you are an expert. This damage increases to 3 if you're a master and 4 if you're legendary.</p><h2 class=\"title\">Master Spellcaster<span style=\"float:right\">Level 15</span></h2><p>You've achieved mastery over your patron's magic. Your proficiency ranks for spell attacks and spell DCs for your witch spellcasting increase to master.</p><h2 class=\"title\">Will of the Pupil<span style=\"float:right\">Level 17</span></h2><p>Communion with your familiar has steeled your mental fortitude. Your proficiency rank for Will saves increases to master. When you roll a success on a Will save, you get a critical success instead.</p><h2 class=\"title\">Legendary Spellcaster<span style=\"float:right\">Level 19</span></h2><p>You've perfected your command of the magic your patron provides. Your proficiency ranks for spell attacks and spell DCs for your witch spellcasting increase to legendary.</p><h2 class=\"title\">Patron's Gift<span style=\"float:right\">Level 19</span></h2><p>Your patron grants you the power to command incredible works of magic. You gain a single 10th-level spell slot and can prepare a spell in that slot using witch spellcasting. Unlike with other spell slots, these spell slots can't be used for abilities that let you cast spells without expending spell slots or abilities that give you more spell slots. You don't gain more 10th-level spells as you level up, though you can take the @UUID[Compendium.pf2e.feats-srd.Item.Patron's Truth] feat to gain a second slot.</p><h2 class=\"title\">Patrons</h2><p>When your patron first makes contact, they usually reveal little more of themself than glimpses or specific manifestations—you might see eyes from the shadows of the forest or hear a wordless voice echoing across the frozen tundra before your familiar makes itself known. Your patron's true identity might be a powerful hag, a fey lord, a deity or demigod, an archdevil, or a similarly powerful entity, or perhaps multiple such figures working in tandem, but that identity is for you to attempt to discover as you learn more of your patron's lessons. Some patrons are multifaceted, appearing in different guises, and granting different powers to different witches. In some cases, the witch may know of their patron's identity, but have some other aspect shrouded in uncertainty; they might know their patron's name but not motive, or perhaps even if they know the patron's identity, they have been cursed to never reveal it to others. </p><p>Your patron determines the following. </p><p><strong>Spell List</strong> You use this magical tradition and spell list; </p><p><strong>Patron Skill</strong> You become trained in the skill associated with your patron's magical tradition. </p><p><strong>Initial Lesson</strong> This is the first lesson your patron teaches you and is unique to your patron. </p><p><strong>Familiar Ability</strong> One of your familiar's two bonus abilities is always the one listed here, a mark of your patron's indelible influence. The benefit can occur only once per round when you Cast or Sustain a hex, and you can choose whether it occurs before or after the effects of Casting or Sustaining the hex.</p><p>@UUID[Compendium.pf2e.classfeatures.Item.Faith's Flamekeeper]</p><p>@UUID[Compendium.pf2e.classfeatures.Item.The Inscribed One]</p><p>@UUID[Compendium.pf2e.classfeatures.Item.The Resentment]</p><p>@UUID[Compendium.pf2e.classfeatures.Item.Silence in Snow]</p><p>@UUID[Compendium.pf2e.classfeatures.Item.Spinner of Threads]</p><p>@UUID[Compendium.pf2e.classfeatures.Item.Starless Shadow]</p><p>@UUID[Compendium.pf2e.classfeatures.Item.Wilding Steward]</p><h2 class=\"title\">Witch Lessons</h2><p>A witch's knowledge from their patron comes in the form of lessons, which you can learn by selecting feats like Basic Lesson. Each lesson grants you a hex and teaches your familiar a new spell, adding it to the spells you can prepare using your witch spellcasting. You gain this spell even if it's not on your tradition's spell list.</p><p><strong>BASIC LESSONS</strong></p><p>You can select from these lessons when a feat or another effect grants you a basic lesson.</p><ul><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Dreams</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Veil of Dreams]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Sleep]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of the Elements</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Elemental Betrayal]</em> hex. Your familiar learns your choice of <em>@UUID[Compendium.pf2e.spells-srd.Item.Breathe Fire]</em>, <em>@UUID[Compendium.pf2e.spells-srd.Item.Air Bubble]</em>, <em>@UUID[Compendium.pf2e.spells-srd.Item.Hydraulic Push]</em>, or <em>@UUID[Compendium.pf2e.spells-srd.Item.Pummeling Rubble]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Life</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Life Boost]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Spirit Link]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Protection</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Blood Ward]</em> hex; your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Mystic Armor]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Vengeance</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Needle of Vengeance]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Phantom Pain]</em>.</li></ul><p><strong>GREATER LESSONS</strong></p><p>You can select from these lessons when a feat or other effect grants you a greater lesson.</p><ul><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Mischief</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Deceiver's Cloak]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Mad Monkeys]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Shadow</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Malicious Shadow]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Chilling Darkness]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Snow</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Personal Blizzard]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Wall of Wind]</em>.</li></ul><p><strong>MAJOR LESSONS</strong></p><p>You can select from these lessons when a feat or other effect grants you a major lesson.</p><ul><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Bargains</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Over the Coals]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Subconscious Suggestion]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Death</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Curse of Death]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Raise Dead]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of Renewal</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Restorative Moment]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Field of Life]</em>.</li><li><span style=\"text-decoration:underline\"><span style=\"text-decoration:underline\">Lesson of the Frozen Queen</span></span>: You gain the <em>@UUID[Compendium.pf2e.spells-srd.Item.Glacial Heart]</em> hex, and your familiar learns <em>@UUID[Compendium.pf2e.spells-srd.Item.Wall of Ice]</em>.</li></ul>",
```